.argo/argo-checkout.yaml

```diff
@@ -3,24 +3,19 @@ type: container
 version: 1
 name: argo-checkout
 description: Checks out a source repository to /src
-
+resources:
```

Comment:
Could you move this after the inputs/outputs sections?
I think it is easier to read if the inputs and outputs are listed first.

Reply:
done